### PR TITLE
create button component, props need changing to allow component reuse

### PR DESCRIPTION
**Pull request by TG-hub**
Haven't quite managed to get the button to be properly reusable

### CODE REVIEW

File: src/components/button.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react'
+import {styles} from './input'
+import Input, {input} from './input'
```
Comment (JohnAGill) [inline src/components/button.js]: Don't think you need these?

File: src/components/button.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react'
+import {styles} from './input'
```
Comment (JohnAGill) [inline src/components/button.js]: each component should have their own styles in a css/scss file

File: src/components/button.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import Input, {input} from './input'
+
+
+function Button (props) {
+    const [submit, setSubmit] = useState('')
```
Comment (JohnAGill) [inline src/components/button.js]: rather than controlling the submit state inside the button could we just pass the onPress as a prop?

**Comment by TG-hub** [comment]
Here's a screenshot, still with borders for the time being just for my own clarity, I'll get rid once I add a background 
Screenshot 2020-09-11 at 16 15 52

File: src/components/input.js [hunk 4]
```diff
@@ -1,22 +1,24 @@
...
-
-    const style = {fontFamily: 'Arial',
+ export const styles = {fontFamily: 'Arial',
         fontStyle: 'normal',
         fontWeight: 'bold',
         fontSize: 16,
         borderRadius: 10,
         color: '#0569FF'}
 
+function Input(props) {
```
Comment (JohnAGill) [inline src/components/input.js]: swap to arrow function please (Didn't see this at first)

File: src/components/button.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react'
+import {styles} from './input'
+import Input, {input} from './input'
+
+
+function Button (props) {
```
Comment (JohnAGill) [inline src/components/button.js]: Arrow functions please

File: src/components/input.js [hunk 6]
```diff
@@ -1,22 +1,24 @@
...
+    const [input, setInput] = useState('')
```
Comment (JohnAGill) [inline src/components/input.js]: These should be handled out side the component and passed in as props

**Review by JohnAGill — CHANGES_REQUESTED** [review]

**Pull request merged**